src/Hybrid/time_triggered.jl

```diff
@@ -216,7 +216,7 @@ function solve(ivp::IVP{<:HACLD1}, args...; kwargs...)
     sol = post(alg, prob, no_tspan; NSTEPS=NHIGH)
 
     if max_jumps == 0
-        return ReachSolution(Flowpipe(sol[1:NLOW-1]), alg)
+        return ReachSolution(Flowpipe(sol), alg)
```

Comment:
the reason behind this choice is that returning the reach-sets with indices NLOW up to NHIGH would break the assumption that `tspan(sol)` contains all possible trajectories of the system within that time-span.

this short time span should give an indication that at least one jump is needed to get the behaviors before and after the jump.

cc @dfcaporale

Reply:
I think if you specify `max_jumps == 0`, you still want to compute the whole flowpipe and not just stop once a guard is enabled.

Reply:
yeah this is in some sense arbitrary. alright if you think that is the reasonable solution, but we should document that behavior (i can do it in another PR).

Reply:
moreover, if we ask for max_jumps = k, we are ATM returning up to NHIGH so it's the same problem.

Reply:
what i dislike is that for example you compare with simulations for the `tspan(sol)` and you'll see some trajectories that go out of the flowpipe.

Reply:
> moreover, if we ask for max_jumps = k > 1, we are ATM returning up to NHIGH so it's the same problem.

Yes, I also wanted to make it consistent with that.

Reply:
=> proposal: store only up to `NLOW - 1` for the last returned flowpipe, for any max_jump >= 0 , and store the slice from NLOW to NHIGH ("non-guaranteed flowpipe") solution in the extension field of the flowpipe.

Reply:
=> the change needed would be after the loop in `@inbounds for k in 2:max_jumps+1` (i didn't try it):

```julia
F_last_jump = out[end][NLOW:NHIGH]
resize!(out[end], 1:NLOW-1)
ReachSolution(HybridFlowpipe(out, Dict("Last jump"=> F_last_jump), alg)
```

Reply:
Do you want to propose a new PR or modify this one? I got lost :smile:

Reply:
if you want to modify this PR with the proposal please go ahead.

otherwise i will do it -- but only after having updated the other open PRs so it will take one day or two.

Reply:
> otherwise i will do it -- but only after having updated the other open PRs so it will take one day or two.

Sure, this is not urgent.

Reply:
Here is a result for `max_jumps=0` where you see that the flowpipe is not fully covered with the current implementation:

![simple_example_det_set_continuous](https://user-images.githubusercontent.com/9656686/81557610-8c7e1880-938c-11ea-83cb-051597d85856.png)

Reply:
Good catch Christian. We missed this edge case.

Reply:
The point that i was making before was that i'd to get `0 .. 0.9` for the `tspan` of the returned flowpipe (this is in fact all that you can guarantee if there is no jump).